addons/sourcemod/scripting/MyJailbreak/dealdamage.sp

```diff
@@ -1189,7 +1201,10 @@ void PrepareDay(bool thisround)
 			MyIcons_BlockClientIcon(i, true);
 		}
 
-		CreateTimer (1.1, Timer_SetModel, i);
+		if (gc_bModel.BoolValue)
+		{
+			CreateTimer (1.1, Timer_SetModel, i);
```

Comment:
We should pass the client userid. And maybe in the callback check if the player still alive or not?

Reply:
I missed that. Yes always pass userid on timer